extractor/src/main/scala/org/jetbrains/sbt/extractors/DependenciesExtractor.scala

```diff
@@ -174,16 +269,14 @@ object DependenciesExtractor extends SbtStateOps with TaskOps {
         classpathConfiguration <- classpathConfigurationTask
       } yield {
 
-        val projectToTransitiveDependencies = if (options.insertProjectTransitiveDependencies) {
-          getTransitiveDependenciesForProject(
-            projectRef,
-            allConfigurationsWithSourceOfAllProjects,
-            classpathConfiguration,
-            settings,
-            buildDependencies
-          )
-        } else
-          Map.empty[ProjectRef, Seq[Configuration]]
+        val arguments = (projectRef, projectToConfigurations, classpathConfiguration, settings, buildDependencies)
```

Comment:
This is an unnecessary complication.
We have little duplication here and I don't think that we will have more then 2.
After some time there will be 1 anyway (after prod/test separation will become the default and only behavior)
It's fine to just inline the arguments.